tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -156,6 +156,17 @@
     <PackageReference Include="Serilog.Extensions.Hosting" Version="4.2.0" Condition="'$(TargetFramework)' == 'net6.0'" />
   </ItemGroup>
 
+  <ItemGroup> <!-- Versions below 4.5 did not support netstandard 2.0 -->
+    <PackageReference Include="NLog" Version="4.5.10" Condition="'$(TargetFramework)' == 'netcoreapp2.1'" />
+    <PackageReference Include="NLog" Version="4.5.9" Condition="'$(TargetFramework)' == 'netcoreapp2.2'" />
+    <PackageReference Include="NLog" Version="4.6.8" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
+    <PackageReference Include="NLog" Version="5.0.0-rc2" Condition="'$(TargetFramework)' == 'net5.0'" />
```

Comment:
Maybe use NLog 5.0 RTM ?

Reply:
Thank you for letting us know that its out!  I will get this updated.

Reply:
Looks like 5.0 was released about the same day as the original PR so makes sense might get missed. Would agree that using the non RC would be ideal, if possible, for verification of the real version. That being said, I'd be fine with that as a follow-up PR.